contracts/WildcardSteward_v2.sol

```diff
@@ -574,7 +739,7 @@ contract WildcardSteward_v2 is Initializable {
             currentOwner,
             tokenPatron,
             address(this),
-            0
+            10**17
```

Comment:
Shouldn't this just be the same as the current price; I don't think this should be hardcoded. If we want to optimize gas usage, we will actually remove this variable from this function and not change it at all when foreclosing.

Reply:
Yeah totally, I've chnaged it price[tokenId] for the auction function :)